static/js/theme-map/SearchDebouncer.js

```diff
@@ -71,7 +71,10 @@ class SearchDebouncer {
    * @returns {number}
    */
   _calculateRelativeDistance (distance, zoom) {
-    return distance * Math.pow(2, zoom - 10);
+    const adjustment = 0.835; // The adjustment ensures that distanceInPixels is accurate
```

Comment:
Just curious: Did you calculate this number or did we just find this gave us the distance we want?

Reply:
I printed out the actual pixel distance and `distanceInPixels` and I found that it was always off by this amount, regardless of the zoom level. So it was calculated based on observation